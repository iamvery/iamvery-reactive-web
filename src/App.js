import React, { Component } from 'react';
import Posts from './Posts';
import './App.css';

class App extends Component {
  render() {
    return (
      <Posts data={['First!', 'Another!']} />
    );
  }
}

export default App;
