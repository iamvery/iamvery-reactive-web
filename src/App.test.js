import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import App from './App';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
});

it('greets the world with a hello', () => {
  const app = shallow(<App />);
  expect(app.text()).toEqual('Hello, World!');
});
