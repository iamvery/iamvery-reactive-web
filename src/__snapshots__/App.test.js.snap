// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`does not change unexpectedly 1`] = `
<div>
  Hello, World!
</div>
`;
